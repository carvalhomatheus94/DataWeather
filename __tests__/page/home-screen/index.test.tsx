import React from 'react'
import renderer from 'react-test-renderer'
import HomeScreen from '../../../src/page/home-screen'

describe('Home test', () => {
  test('testing if renders correctly', () => {
    const tree = renderer
      .create(
        <HomeScreen />,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
