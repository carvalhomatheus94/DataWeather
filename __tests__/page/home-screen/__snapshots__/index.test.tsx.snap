// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home test testing if renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      Object {
        "backgroundColor": "#333",
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "flex": 1,
          "justifyContent": "space-between",
        },
      ]
    }
  >
    <View
      style={
        Object {
          "padding": 16,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#fff",
            "fontSize": 18,
            "fontWeight": "bold",
          }
        }
      >
        Atualize os dados da sua cidade
      </Text>
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#000",
          "height": 48,
          "justifyContent": "center",
          "opacity": 1,
          "width": "100%",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#939AA4",
            "fontSize": 18,
            "textTransform": "uppercase",
          }
        }
      >
        Atualizar Busca
      </Text>
    </View>
  </View>
</View>
`;
